components/src/histogram.tsx

```diff
@@ -19,18 +19,27 @@
  *
  */
 
-import iobioCharts from 'iobio-charts';
-import { useEffect } from 'react';
+import 'iobio-charts';
 
-const { createHistogram } = iobioCharts;
-
-function Histogram(props: { 'broker-key': string }) {
-	useEffect(() => {
-		createHistogram();
-	}, []);
-
-	// return <iobio-histogram />;
-	return <div broker-key={props['broker-key']}></div>;
+function IobioHistogram({
+	brokerKey,
+	color,
+	ignoreOutliers,
+	title,
+}: {
+	brokerKey: string;
+	color?: string;
```

Comment:
If we expect we'll be exposing multiple styling parameters, maybe you'd want to create a `style` object to contain all of these?

I imagine we'll eventually want to control styles across the whole app through some sort of ThemeProvider... but thats for future work.

```ts
type IobioHistorgramProps = {
	brokerKey: string;
	ignoreOutliers?: boolean;
	style?: {
		color?: string;
	}
	title?: string;
}

function IobioHistogram({
	brokerKey,
	style,
	ignoreOutliers,
	title,
}: IobioHistorgramProps) { /* ... */ }
```

Reply:
So I discussed this with Anders yesterday, the web components need parent containers with explicit dimensions and their recommended approach is adding style definitions on the container.

I tested the color variable and it works this way. The components also contain classes (i.e. `.iobio-title`) we can leverage as selectors.

So maybe that will suffice and a style/color prop may not be needed

Reply:
Isn't the shadow DOM locally scoped? 
I think the colours work because they're css variables which bleed through shadow DOM
(API has definitely changed since I last looked at it, so I'm not entirely sure)

Reply:
CSS variables are inheritable, so a style property or class on the parent works fine if it's modifying the color variable

I will revisit tomorrow how we can add styles to elements like the title etc., I've tried a few workarounds and may need to reconnect with the Iobio team